gcp/modules/gpii-flowmanager/resources/flowmanager-network-policy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: flowmanager-network-policy
+  namespace: gpii
+spec:
+  podSelector:
+    matchLabels:
+      app: flowmanager
+  policyTypes:
+  - Egress
+  - Ingress
```

Comment:
Is the 'Ingress' needed if no ingress block defined?

Reply:
I think the intent here is to use an empty Ingress to block all (non-specified) ingress traffic: https://kubernetes.io/docs/concepts/services-networking/network-policies/#default-deny-all-ingress-traffic